packages/chevrotain/api.d.ts

```diff
@@ -882,6 +882,21 @@ export declare class Parser {
         options: AtLeastOneSepMethodOpts<any>
     ): void
 
+    /**
+     *
+     * @param name - The name of the rule.
```

Comment:
We should avoid duplicating the documentation.
See:  the "@see" annotations used a little above  in this file.

Only the return type docs should be different.